tables/antibodies.csv

```diff
@@ -3392,6 +3392,7 @@ EY6A,NULL,NULL,NULL,Chang Gung Memorial Hospital,CP,5000,FALSE,2021-04-27
 Ehling_mAb-50,NULL,NULL,NULL,NULL,CP,1000,FALSE,2022-02-24
 Ehling_mAb-64,NULL,NULL,NULL,NULL,CP,1000,FALSE,2022-02-24
 Ehling_mAb-82,NULL,NULL,NULL,NULL,CP,1000,FALSE,2022-02-24
+Ensitrelvir,NULL,NULL,NULL,NULL,NULL,5000,FALSE,2023-03-21

```

Comment:
Remdesivir, Molnupiravir, Nirmatrelvir, and Ensitrelvir are not antibodies. Instead of `tables/antibodies.csv` and `tables/rx_antibodies`, they should be stored in `tables/compounds.csv` and `tables/rxtables/rx_compounds`.